mne_bids/mne_bids.py

```diff
@@ -67,6 +69,18 @@ def _channels_tsv(raw, fname, verbose):
                     eog='ElectrOculoGram', misc='Miscellaneous',
                     ref_meg='Reference channel')
 
+    # get the manufacturer from the file in the Raw object
+    if hasattr(raw, 'filenames'):
+        _, ext = _parse_ext(raw.filenames[0], verbose=verbose)
+        manufacturer = MANUFACTURERS[ext]
+    else:
+        manufacturer = None
+
+    ignored_indexes = list()
+    for ch_name in IGNORED_CHANNELS.get(manufacturer, list()):
```

Comment:
simply `IGNORED_CHANNELS[manufacturer]` ?

Reply:
if the manufacturer isn't in the IGNORED_CHANNELS dictionary this would give a `KeyError`. But it can be modified by making it a one-liner as you did below.